src/azure-cli/azure/cli/command_modules/storage/operations/blob.py

```diff
@@ -307,11 +304,66 @@ def set_service_properties(client, parameters, delete_retention=None, delete_ret
     return client.get_blob_service_properties()
 
 
-def storage_blob_copy_batch(cmd, client, source_client, container_name=None,
-                            destination_path=None, source_container=None, source_share=None,
-                            source_sas=None, pattern=None, dryrun=False):
-    """Copy a group of blob or files to a blob container."""
+# pylint: disable=too-few-public-methods
+class BlobServiceProperties:
```

Comment:
Is there a corresponding class in SDK?

Reply:
`get_service_properties()` returns a dict and I need to create a generic update command. It seems that generic update command only accept instance as input, so I convert it to `BlobServiceProperties`.